automation-tests/.vscode/tasks.json

```diff
@@ -0,0 +1,27 @@
+{
+  // See https://go.microsoft.com/fwlink/?LinkId=733558
+  // for the documentation about the tasks.json format
+  "version": "2.0.0",
+  "tasks": [
```

Comment:
These weren't showing in my Task dropdown - is there anything else I needed to do to get them to display? 

<img width="877" alt="Screenshot 2023-01-20 at 3 37 55 PM" src="https://user-images.githubusercontent.com/9795193/213800981-6f43db4d-4c28-41c1-b4bd-181592678460.png">

Reply:
They're now appearing - I was likely in the wrong directory while trying to view these